API Blueprint Specification.md

```diff
@@ -478,6 +490,232 @@ Name and description of the API
+        + access_token_endpoint: `/token`
+        + token_header_keyword: `token`
+        + scopes: read, write, email
+
+#### Bearer Authentication Scheme
+This scheme represents the method of sending a bearer token as specified in [RFC 6750](https://tools.ietf.org/html/rfc6750#section-2.1).
+
+The customizable options for this scheme are:
+
++ `keyword` represents the string used instead of "Bearer" in the "Authorization" header field according to [RFC 6750](https://tools.ietf.org/html/rfc6750#section-2.1). Value type is **string**. "Bearer" is the **default**.
++ `example_token` represents an example bearer token value. Value type is **string**. "example_token" is the **default**.
+
+##### Example
+
+    + token (Bearer Authentication Scheme)
+        + keyword: `token`
+
+---
+
+## Authenticated section
+- **Parent sections:** [Resource section](#def-resource-section), [Action section](#def-action-section), [Request section](#def-request-section)
+- **Nested sections:** none
+- **Markdown entity:** list
+- **Inherits from**: none
+
+#### Definition
+Defined by the `Authenticated` keyword in Markdown list entity followed by optional comma seperated list of scheme names which are prefixed by a colon.
+
+    + Authenticated: <list>
+
+> **NOTE:** If name of an authentication scheme dervied from `OAuth2 Authentication Scheme` is used, it can optionally have comma seperated list of oauth scopes inside `[]` immediately following it.
+
+#### Description
+This section describes an authentication setting. Based on the parent section, the authentication setting being described is one of the following:
+
+1. Resource authentication setting ([Resource section](#def-resource-section))
+2. Action authentication setting ([Action section](#def-action-section))
+3. Request authentication setting ([Request section](#def-request-section))
+
+The authentication setting for a [Request section](#def-request-section) denotes the authentication needed to be sent along with the request to get the [Response section](#def-response-section) of the respective transaction example.
+
+If the authentication setting contains only the `Authenticated` keyword, the transaction example allows all the authentication schemes described in the [Authentication schemes section](#def-authenticationschemes-section).
+
+```
++ Authenticated
+```
+
+If the authentcation setting contains a list of scheme names after the `Authenticated` keyword, the transaction example only allows the specific authentication schemes denoted by the scheme names.
+
+```
++ Authenticated: oauth, token
+```
+
+```
++ Authenticated: oauth[read, email], token
+```
+
+#### Resource Authenticated description
+Description of the resource authentication setting.
+
+If defined in a [Resource section](#def-resource-section), all the requests of the transaction examples under this resource will inherit the authentication setting unless specified otherwise.
+
+##### Example
+
+    # User [/user]
+
+    + Authenticated
```

Comment:
If this is oauth, it might be nice to describe this group or resource requires a particular oauth scope, along with how they are inherited (or overridden). (linking back to https://github.com/apiaryio/api-blueprint/pull/201/files?short_path=c4fbef2#r35647854)

```
+ Authenticated: oauth[user.read]
```